src/main/java/seedu/address/logic/commands/RiskProfileCommand.java

```diff
@@ -0,0 +1,224 @@
+            personToEdit.getAddress(), personToEdit.getApptDate(), riskProfile, personToEdit.getTags());
+
+        model.setPerson(personToEdit, editedPerson);
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+
+        return new CommandResult(generateSuccessMessage(editedPerson));
+    }
+
+    /**
+     * Generates a command execution success message when
+     * the risk profile is added.
+     * {@code personToEdit}.
+     */
+    private String generateSuccessMessage(Person personToEdit) {
+        return String.format(MESSAGE_ADD_RISKPROFILE_SUCCESS, personToEdit);
+    }
+
+    /**
+     * Calculates the total score from the result.
+     *
+     * @param result The result of risk profile.
+     * @return The total score calculated from the result.
+     */
+    public static int calculateTotalScore(String result) {
+        int countA = 0;
+        int countB = 0;
+        int countC = 0;
+        int countD = 0;
+        int countE = 0;
+
+        for (char ch : result.toCharArray()) {
+            switch (ch) {
+            case 'a':
+                countA++;
+                break;
+            case 'b':
+                countB++;
+                break;
+            case 'c':
+                countC++;
+                break;
+            case 'd':
+                countD++;
+                break;
+            case 'e':
+                countE++;
+                break;
+            default:
+                break;
+            }
+        }
+
+        return countA + countB * 2 + countC * 3 + countD * 4 + countE * 5;
+    }
+
+    /**
+     * Calculates the risk level based on the total score.
+     *
+     * @param totalScore The total score calculated from the result.
+     * @return The risk level determined from the total score.
+     */
+    public static String calculateRiskLevel(int totalScore) {
+        String result;
+        switch (totalScore) {
+        case 1:
+        case 2:
+        case 3:
+        case 4:
+        case 5:
+        case 6:
+        case 7:
+        case 8:
+            result = "Low";
+            break;
+        case 9:
+        case 10:
+        case 11:
+        case 12:
+        case 13:
+        case 14:
+        case 15:
+        case 16:
+            result = "Moderately Low";
+            break;
+        case 17:
+        case 18:
+        case 19:
+        case 20:
+        case 21:
+        case 22:
+        case 23:
+        case 24:
+            result = "Moderate";
+            break;
+        case 25:
+        case 26:
+        case 27:
+        case 28:
+        case 29:
+        case 30:
+        case 31:
+        case 32:
+            result = "Moderately High";
+            break;
+        case 33:
+        case 34:
+        case 35:
+        case 36:
+        case 37:
+        case 38:
+        case 39:
+        case 40:
+            result = "High";
```

Comment:
Why not use if statement instead? this switch statement looks so long

Reply:
Thank you for the suggestion.